src/test/kotlin/lotto/model/LottoTest.kt

```diff
@@ -3,64 +3,57 @@ package lotto.model
 import org.junit.jupiter.api.Assertions.assertTrue
+import org.junit.jupiter.api.Test
 import org.junit.jupiter.api.assertDoesNotThrow
 import org.junit.jupiter.api.assertThrows
 import org.junit.jupiter.params.ParameterizedTest
-import org.junit.jupiter.params.provider.CsvSource
 import org.junit.jupiter.params.provider.ValueSource
 import kotlin.IllegalArgumentException
 
 class LottoTest {
-    @ParameterizedTest
-    @ValueSource(strings = ["1, 2, 3, 4, 5", "1, 2, 3, 4, 5, 6, 7"])
-    fun `로또 번호는 6개의 자연수를 갖지 않으면 오류를 발생시킨다`(input: String) {
-        val numbers = input.split(", ").map { LottoNumber.valueOf(it.toInt()) }.toSet()
-        assertThrows<IllegalArgumentException> { Lotto(numbers) }
+    @Test
+    fun `로또 번호의 수가 6개 미만일 경우, 오류를 발생시킨다`() {
+        assertThrows<IllegalArgumentException> { Lotto.of(1, 2, 3, 4, 5) }
     }
 
-    @ParameterizedTest
-    @ValueSource(strings = ["1, 1, 1, 2, 3, 4", "1, 2, 3, 4, 5, 1", "1, 1, 1, 1, 1, 1"])
-    fun `로또 번호는 중복 번호가 포함될 경우 오류를 발생시킨다`(input: String) {
-        val numbers = input.split(", ").map { LottoNumber.valueOf(it.toInt()) }.toSet()
-        assertThrows<IllegalArgumentException> { Lotto(numbers) }
+    @Test
+    fun `로또 번호의 수가 6개를 초과할 경우, 오류를 발생시킨다`() {
+        assertThrows<IllegalArgumentException> { Lotto.of(1, 2, 3, 4, 5, 6, 7) }
     }
 
-    @ParameterizedTest
-    @ValueSource(strings = ["1, 2, 3, 4, 5, 6", "40, 41, 42, 43, 44, 45", "7, 2, 3, 4, 5, 35"])
-    fun `로또 번호는 6개의 서로 다른 자연수를 갖는다`(input: String) {
-        val numbers = input.split(", ").map { LottoNumber.valueOf(it.toInt()) }.toSet()
-        assertDoesNotThrow { Lotto(numbers) }
+    @Test
+    fun `로또 번호는 중복 번호가 포함될 경우 오류를 발생시킨다`() {
+        assertThrows<IllegalArgumentException> { Lotto.of(1, 1, 1, 1, 1, 2) }
     }
 
-    @ParameterizedTest
-    @CsvSource(
-        "1 2 3 4 5 6,6",
-        "40 41 42 43 44 45,0",
-        "7 2 3 4 5 35,4",
-    )
-    fun `당첨 번호를 비교하여 동일한 번호의 개수를 반환한다`(
-        winning: String,
-        expected: Int,
-    ) {
+    @Test
+    fun `로또 번호는 6개의 서로 다른 자연수를 갖는다`() {
+        assertDoesNotThrow { Lotto.of(1, 2, 3, 4, 5, 6) }
+    }
+
+    @Test
+    fun `당첨 번호를 비교하여 동일한 번호의 개수를 반환한다`() {
         val lotto = Lotto.of(1, 2, 3, 4, 5, 6)
-        val winningNumbers = winning.split(" ").map { LottoNumber.valueOf(it.toInt()) }.toSet()
-        val result = lotto.countMatchingNumbers(Lotto(winningNumbers))
+        val winningNumbers = Lotto.of(1, 2, 3, 7, 8, 9)
+        val result = lotto.countMatchingNumbers(winningNumbers)
+        val expected = 3
+
         assertThat(result).isEqualTo(expected)
     }
 
     @ParameterizedTest
     @ValueSource(ints = [1, 5, 6])
     fun `로또 번호에 보너스 번호가 포함되어 있으면 True를 반환한다`(bonusNumber: Int) {
         val lotto = Lotto.of(1, 2, 3, 4, 5, 6)
-        val result = lotto.hasMatchingBonusNumbers(LottoNumber.valueOf(bonusNumber))
+        val result = lotto.hasMatchingBonusNumbers(LottoNumber.valueOf(bonusNumber)!!)
         assertTrue(result)
```

Comment:
아래 코드와 어떤 차이점이 생겨날까요 ?_?
```suggestion
        val result = lotto.hasMatchingBonusNumbers(LottoNumber.valueOf(bonusNumber))
        assertThat(result).isTrue()
```

Reply:
`assertThat(result).isTrue()` 
훨씬 가독성이 좋은 것 같습니다! 😄

Reply:
result가 `non-null` -> `nullable` 이 되는 변화점에서도 더 생각해보세요!

Reply:
`!!` 사용하는 게 조금 찜찜했었는데, 
`contains` 의 인자로 nullable 값도 들어갈 수 있다는 걸 알게되었네욤..!